src/api/layers/GeoAdminWMSLayer.class.js

```diff
@@ -79,6 +79,12 @@ export default class GeoAdminWMSLayer extends GeoAdminLayer {
         this.wmsVersion = wmsVersion
     }
 
+    clone() {
+        const clone = super.clone()
+        clone.timeConfig = this.timeConfig ? this.timeConfig.clone() : null
```

Comment:
This line could be simplified to `clone.timeConfig = this.timeConfig?.clone() 
?? null`